docs/source/guides/install/docker.rst

```diff
@@ -2,141 +2,92 @@
 Docker
 ======
 
-For automating ``subscriptions.yaml`` downloads to pull new media, see :ref:`this page <guides/getting_started/automating_downloads:docker and unraid>` on how to set up a cron job in any of the docker containers.
-
-The ``ytdl-sub`` Docker images use :lsio:`LSIO-based images <\ >` and install ytdl-sub on top. There are two flavors to choose from.
-
-.. margin:: 
-
-  .. tip:: 
-
-    The recommended docker image is the GUI image.
-
-    :ref:`Docker Compose <guides/install/docker:install with docker compose>` is the recommended way of setting up a ``ytdl-sub`` docker container.
+The ``ytdl-sub`` Docker images use :lsio:`LSIO-based images <\ >` and install ytdl-sub
+on top. There are two flavors or variants to choose from. For a more user-friendly
+experience editing the `configuration`_, we recommend the `GUI image`_
+variant. :ref:`Docker Compose <guides/install/docker:install with docker compose>` is
+the recommended way of managing a ``ytdl-sub`` docker container.  See :ref:`Automating
+Downloads <guides/getting_started/automating_downloads:docker and unraid>` for how to
+automate running ``ytdl-sub`` in a container running either variant.
 
 GUI Image
 ---------
 
-The GUI image uses LSIO's :lsio-gh:`docker-code-server image <\ >` for its base image. More info on other code-server environment variables can be found within its documentation.
+The GUI image is based on LSIO's :lsio-gh:`docker-code-server`. See its documentation
```

Comment:
Let's add something along the lines of:

"based on LSIO's docker code-server. This provides full management of ytdl-sub (file editing, terminal access) all within the browser using code-servers' browser-based VS Code."

To better express the novelty and difference of the GUI image.

Reply:
Excellent useful additional context and something I just couldn't write having not used it. Thanks and I'll add that.